### ISSUE TITLE
Add keywords to PanDataset

### ISSUE THREAD

**Issue by fspreck-indiscale** [pangaea-data-publisher/pangaeapy#56]
It would be good to have the keywords (actual, not technical or auto-generated) in the dataset object, too.  This could be a simple list of keyword strings, but could also be a list of dictionaries.  E.g., for https://doi.pangaea.de/10.1594/PANGAEA.957810, it could be either

```
["Benguela Upwelling System", "elemental stoichiometry", "marine carbon cycle", "Sea surface partial pressure of CO2"]
```

or 

```
[
  {
    "name": "Benguela Upwelling System",
    "id":  "keywords.term72123",
    "type": "fromDatabase"
  },
  ...
]
```
The former is a bit easier to read while the latter is more easy to extend, so I've a slight tendency toward the list of dictionaries.

@huberrob Which way would you prefer?  I'm happy to provide a pull request for the implementation.

**Comment by huberrob** [comment]
Yes of course, please feel free to add this. A simple list would be enough

**Comment by fspreck-indiscale** [comment]
Ok, then there is the PR for simple lists in #57 :-)

**Comment by fspreck-indiscale** [comment]
Implemented in #57.
